catsgo.py

```diff
@@ -32,6 +33,7 @@ def login():
     try:
         load_cookies()
     except:
+        logging.error(f'''Login {traceback.format_exc()}''')
```

Comment:
We would usually use single (') or double quoted (") format strings. Not sure what's best, but we should be consistent when triple quoted isn't NEEDED.

Reply:
This probably should be done too